setup.py

```diff
@@ -1,7 +1,7 @@
 import os
 from setuptools import setup, find_packages
 
-VERSION = '0.1.11'
+VERSION = '0.1.12'
```

Comment:
Can we also bump the schema in the ReadMe with a description of what this change does please?